src/components/startups/Welcome.vue

```diff
@@ -221,6 +220,14 @@ export default {
 
 <style>
 
+.kiwi-startup-common-section.kiwi-startup-common-section-connection {
```

Comment:
All of these `kiwi-startup-common-` CSS selectors should not be in here as this is the Welcome component. A perfect example on why this is bad though - these width rules are now conflicting with the styles in the `src/components/startups/CommonLayout` component. All of this `kiwi-startup-common-` styling should be in that CommonLayout component and you will then see the conflicting styles right away.